### ISSUE TITLE
OKS Pioneer, Kerbitat, Workshop missing MKSModule

### ISSUE THREAD

**Issue by tarheel1999** [BobPalmer/MKS#882]
The OKS Pioneer, Kerbitat, and Workshop are missing the MKSModule.  This negatively impacts the efficiency of these parts and prevents Kolony Rewards being granted to stations consisting solely of any of these parts.

I'm not sure what balance you are striving for but OKS Aeroponics has a workspace of 1 and a living space of 0. The MKV Workshop has a work space of 4 and a living space of 1.  The MK3 Fabricator has a workspace of 5 and a living space of 0.  The MK3 also has efficiency parts which the OKS Workshop could probably use (thinking of the OKS Workspace here).  

See below:

OKS Aeroponics
 
> MODULE
> 	{
> 		name = MKSModule
> 		workSpace = 1
> 		livingSpace = 0
> 		efficiencyPart = OKS_AgModule,5,USILS_Greenhouse_LG,4
> 	}
 
MKV Workshop

> MODULE
> 	{
> 		name = MKSModule
> 		workSpace = 4
> 		livingSpace = 0
> 		CrewBonus = .5
> 		MaxEfficiency = 1
> 	}

MK3 Fabricator

> MODULE
> 	{
> 		name = MKSModule
> 		workSpace = 5
> 		livingSpace = 0
> 		efficiencyPart = MKV_Workshop,1,OKS_Workspace,1	
> 	}

**Comment by Businfu** [comment]
I'm not sure if I should open an entirely new issue for this so I'm starting off by tagging it on here.  There are other parts (besides the ones listed) which don't have MKSModule, and IMHO it makes determining which parts will (and won't) give Kolony Rewards confusing.  

I have to preface by saying that my original (mis)understanding was that manned UKS parts would trigger Kolony Rewards.  After looking into it, I realized that this isn't the case. 

For example, MKS_ColonyHub (UKS Pioneer Module (Surface)) does not count as a 'base' for Kolony Rewards, despite the fact that it's got a crew capacity of 2 and is described as being 'the first module in place for a budding colony'. In addition to parts already mentioned, the surface Kerbitat and PDU both do not trigger Kolony Rewards.  I didn't look exhaustively at all the configs but there may be more.

In Contrast, the MKV_Lander (MK-V 'Koala' Colony Hub -- the landing legs with KIS storage), which has no crew space, counts as an MKSModule and triggers Kolony Rewards.  Note that the other two base parts (MKS_MobileBase, MKS_ModuleBase) don't have this capability.  I also noticed that the MKV_Sifter (the little radial guy), DOES count as a Kolony all by itself because it has MKSModule. 

Maybe this isn't an issue most people notice because they tend to send up a whole bunch of UKS parts at once, but when you are playing career mode and haven't unlocked parts of the tech tree, it feels inconsistent and a bit unpredictable.  

Since MKSModule affects the workload efficiency of parts, it isn't a trivial thing to just patch it in and out, and thus i'm not entirely sure how to proceed on a PR.

**Comment by SCESW** [comment]
While playing with this mod, I found that I am confused about when a MKS part is compatible with another part and I wondered if I might be able to edit the tooltip/help files included inside.  So, after reading your issue above, I made a little list of parts that currently include "MKSMODULE" and the "efficiencyPart" that is included with each one.  
Although I am not entirely certain what MKSModule does, I printed out this list to use while building bases.

Includes MKSModule:
MK3_Akademy: efficiencyPart = None
MK3_Fabricator: efficiencyPart = MKV_Workshop, OKS_Workspace
MK3_FuelRefinery: efficiencyPart = MKV_CommPak
MK3_Nuke Processor: efficiencyPart = Scientist, Engineer
MK3_Refinery: efficiencyPart = MKV_Smelter, MKV_Sifter, OKS_Workspace
MK3_IndustrialSifter: efficiencyPart = MKV_Smelter, MKV_Sifter, OKS_Workspace, MKV_Crusher
MKS_Aeroponics: efficiencyPart = MKV_AgModule, USILS_Greenhouse_LG
MKS_Workspace: efficiencyPart = none
MKV_AgModule: efficiencyPart = Scientist, Engineer
MKV_Crusher: efficiencyPart = none
MKV_HabModule: efficiencyPart = none
MKV_Lander: efficiencyPart = none
MKV_Pod: efficiencyPart = none
MKV_PowerPack: efficiencyPart = none
MKV_Sifter: efficiencyPart = none
MKV_Smelter: efficiencyPart = none
MKV_Workshop: efficiencyPart = none
USI_MiniRover: efficiencyPart = none
OKS_Aeroponics: efficiencyPart = OKS_AgModule, USILS_Greenhouse_LG
OKS_AgModule: efficiencyPart = Scientist, Engineer
OKS_HabRing: efficiencyPart = none
OKS_Workspace: efficiencyPart = none

_I am relatively new to KSP and have only begun browsing GitHub and this code, so I am not a good person to start offering to 'fix' things.  However, I am good at spotting inconsistencies and compiling lists of my findings.  With continued study, I hope to be able to contribute something useful to KSP in the future._
